product-service-description/compliance-as-a-service/features/basics/tse-as-a-service/deutsche-fiskal.md

```diff
@@ -25,6 +25,10 @@ Das _fiskaltrust.Middleware.SCU.DE.DeutscheFiskal_-Package ermöglicht die Verbi
 | _ProxyPort_ | Falls der Zugang zum Internet über einen Proxy-Server erfolgen muss: Der HTTP-Port des Proxy-Servers. | ja (Default: leer) |
 | _ProxyUsername_ | Falls der Zugang zum Internet über einen _authentifizierten_ Proxy-Server erfolgen muss: Der Benutzername des Proxy-Servers. | ja (Default: leer/keine Proxy-Authentifizierung) |
 | _ProxyPassword_ | Falls der Zugang zum Internet über einen _authentifizierten_ Proxy-Server erfolgen muss: Das Passwort des Proxy-Servers. | ja (Default: leer/keine Proxy-Authentifizierung) |
+| _DontAddFccFirewallException_ | Wenn dieser Parameter nicht gesetzt ist, versucht die SCU beim ersten Start automatisch, eine Firewall-Exception für den FCC anzulegen (dazu werden Admin-Rechte benötigt). Dies kann verhindert werden, indem dieser Parameter auf `true` gesetzt wird. | ja (Default: `false`) |
```

Comment:
Should we add the details for this rule so that IT Admins could extend their documentation?

Reply:
Good point, thanks! Added something to the paragraph in _Troubleshooting_.